VENTouchLock/VENTouchLock.m

```diff
@@ -174,71 +180,57 @@ - (void)requestTouchIDWithCompletion:(void (^)(VENTouchLockTouchIDResponse))comp
 
 - (void)lock
 {
-    if (![self isPasscodeSet]) {
+    if (![self isPasscodeSet] || self.lockWindow.isKeyWindow) {
         return;
     }
 
     if (self.splashViewControllerClass != NULL) {
         VENTouchLockSplashViewController *splashViewController = [[self.splashViewControllerClass alloc] init];
         if ([splashViewController isKindOfClass:[VENTouchLockSplashViewController class]]) {
-            UIWindow *mainWindow = [[UIApplication sharedApplication].windows firstObject];
-            UIViewController *rootViewController = [UIViewController ventouchlock_topMostController];
-            UIViewController *displayController;
+            self.lockWindow = [[UIWindow alloc] initWithFrame:self.mainWindow.frame];
```

Comment:
Up the `UIWindowLevel` doesn't help in the situation i mentioned, by the same token that `UIApplication` is a shared instance. Basically, we want to make sure that we call `[self.lockWindow makeKeyAndVisible];` **last** with respect to whatever the host app might be doing. Hope this makes sense :)

By introducing a delay helps, but not ideal
```
dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(.5 * NSEC_PER_SEC)), dispatch_get_main_queue(), ^{
    [self.lockWindow makeKeyAndVisible];

});
```